assets/src/edit-story/elements/video/test/output.js

```diff
@@ -74,19 +73,8 @@ describe('Video output', () => {
     await expect(<VideoOutput {...props} />).toBeValidAMPStoryElement();
   });
 
-  it('an undefined alt tag in the element should fall back to the resource', async () => {
```

Comment:
Note that now none of the tests check for neither `title` or `alt` anymore.

What I'd expect:

* `it('uses the element's title for the title attribute')`
* `it('should fall back to the resource's title if the element's title is undefined')`
* `it('should not fall back to the resource's title if the element's title is an empty string')`
* `it('should element's title for the alt attribute')`

Reply:
Got it, added those 4 tests.